### ISSUE TITLE
installFeature task fails after version bump to 3.3 plugin with gradle 7.2

### ISSUE THREAD

**Issue by corporate-gadfly** [OpenLiberty/ci.gradle#643]
No issues with 3.2.1. After version bump to 3.3:
```
> Task :installFeature FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':installFeature' (type 'InstallFeatureTask').
  - In plugin 'liberty' type 'io.openliberty.tools.gradle.tasks.InstallFeatureTask' property 'additionalJsonList' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - In plugin 'liberty' type 'io.openliberty.tools.gradle.tasks.InstallFeatureTask' property 'newProject' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#missing_annotation for more details about this problem.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 11s
8 actionable tasks: 8 executed
```

**Comment by corporate-gadfly** [comment]
Looks like the method needs `@Internal` notation @jjiwooLim?

**Comment by cherylking** [comment]
@corporate-gadfly Thanks for bringing this to our attention.

**Comment by corporate-gadfly** [comment]
Not sure why this was closed? Still failing with gradle 7.3/7.4 with liberty-gradle-plugin 3.3.

**Comment by cherylking** [comment]
@corporate-gadfly It was closed because a fix was merged. When the next release is published, it will include this fix.

**Comment by cherylking** [comment]
This fix is in the 3.3.1 release published today @corporate-gadfly

**Comment by corporate-gadfly** [comment]
TY, works nicely with gradle 7.4.2.
